src/Aspirate.Commands/Actions/Manifests/GenerateDockerComposeManifestAction.cs

```diff
@@ -16,17 +19,57 @@ public override Task<bool> ExecuteAsync()
         }
 
         var outputFile = Path.Combine(AspirateLiterals.DefaultOutputPath, "docker-compose.yml");
+        var composeOverrideFile = Path.Combine(AspirateLiterals.DefaultOutputPath, "docker-compose.override.yml");
 
         Logger.MarkupLine($"\r\n[bold]Generating docker compose file: [blue]'{outputFile}'[/][/]\r\n");
 
         var services = new List<Service>();
+        var composeOverride = new List<Service>();
+
+        if (CurrentState.ComposeOverride)
+        {
+            // Add aspire dashboard by default
+            var aspireDashboardService = new Service
```

Comment:
Oh nice - does this work when deployed in compose?
Have they now worked around the issue of requiring the resource server as well?

If so I'd like to remove this from here if possible, and introduce a new action that handles this - as we'd want to plug it into both the k8s and compose generators

Initially you could leave it here and I will pick that up

Reply:
I'll pull the branch tomorrow, and commit the change to make it reusable. Today has been a Little crazy, sorry

Reply:
It only had the logs and metrics, it didn't have resources, so likely this will need to change. When I can, I'll look at how the resource support can be added.. No worries 😄